test/libsolidity/semanticTests/inlineAssembly/constant_access_referencing.sol

```diff
@@ -22,5 +22,7 @@ contract C {
         }
     }
 }
```

Comment:
For completion, can you also add a variable `string constant g = "test"`

Reply:
I don't think this is possible because it is not a value type.

Reply:
Yes, there is a syntax test for that: test/libsolidity/syntaxTests/inlineAssembly/constant_array.sol